### PR TITLE
Histograms

### PR DESCRIPTION
**Pull request by vlandham**
its a start

histograms

todo: 
* improve consistency of labels with other chart areas
* figure interaction
* add axis titles or more labels.

### CODE REVIEW

**Comment by vlandham** [comment]
ping @pbeshai

File: src/components/HistoGroup/HistoGroup.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+   * @param {Object} field Name and id of group of metrics to show in chart
+   * @param {Object} allData Data for the field from summary
+   */
+  renderPlots(field, allData) {
+    const { viewMetric } = this.props;
+    const data = allData ? allData.clientIspsData : [];
+    // console.log(data);
+    const binSets = data.map((d) => d[viewMetric.percentBinKey]);
+    const colors = colorsFor(data, (d) => d.id);
+    // console.log(binSets);
```
Comment (pbeshai) [inline src/components/HistoGroup/HistoGroup.jsx]: log log log

File: src/components/Histogram/Histogram.jsx [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+  /**
+   * Update the d3 chart - this is the main drawing function
+   */
+  update() {
+    const { innerMargin } = this.visComponents;
+    this.g.attr('transform', `translate(${innerMargin.left} ${innerMargin.top})`);
+
+    this.updateAxes();
+    this.updateBars();
```
Comment (pbeshai) [inline src/components/Histogram/Histogram.jsx]: Interesting change from `render` to `update` - thoughts?

File: src/containers/LocationPage/LocationPage.jsx [hunk 3]
```diff
@@ -551,19 +557,14 @@ class LocationPage extends PureComponent {
 
   renderFixedCompareMetrics() {
     const { compareMetrics, summary = {} } = this.props;
-    const fields = [
-      { id: 'lastWeek', label: 'Last Week' },
-      { id: 'lastMonth', label: 'Last Month' },
-      { id: 'lastYear', label: 'Last Year' },
-    ];
```
Comment (pbeshai) [inline src/containers/LocationPage/LocationPage.jsx]: Bravo

File: src/components/Histogram/Histogram.jsx (hunk 2 above)
Comment (vlandham) [inline src/components/Histogram/Histogram.jsx]: I think the use of `update` is consistent with the `update` parent function. And it maintains a naming / logical distinction between "things that react creates" (`render`) and "things that are handled by d3" (`update`).

**Pull request merged**